src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.includes(':'))
+    .reduce((obj, item) => {
+      const [property, ...value] = item.split(':');
+      const trimProperty = property.trim();
+      const trimValue = value.join(':').trim();
+
+      if (trimProperty && trimValue) {
+        obj[trimProperty] = trimValue;
+      }
+
+      return obj;
+    }, {});
```

Comment:
The implementation of the `convertToObject` function is correct and follows best practices for converting a string of CSS styles into an object. The use of `split`, `map`, `filter`, and `reduce` is appropriate here and aligns with the checklist item to avoid loops and use iteration methods instead. However, the variable naming for the accumulator in the `reduce` method (`obj`) could be improved to enhance readability, as per the checklist.